packages/arb-token-bridge-ui/src/components/common/Header.tsx

```diff
@@ -1,378 +1,77 @@
-  {
-    title: 'Arbitrum Goerli',
-    link: getExplorerUrl(ChainId.ArbitrumGoerli)
-  },
-  {
-    title: 'Arbitrum Sepolia',
-    link: getExplorerUrl(ChainId.ArbitrumSepolia)
-  }
-])
-
-const chartsStatsMenuProps = toHeaderMenuProps([
-  {
-    title: 'How much gas am I saving?',
-    link: 'https://gas.arbitrum.io'
-  },
-  {
-    title: 'What’s up with my retryable?',
-    link: 'https://retryable-tx-panel.arbitrum.io'
-  },
-  {
-    title: 'How popular is Arbitrum?',
-    link: 'https://dune.com/Henrystats/arbitrum-metrics'
-  },
-  {
-    title: 'Which L2 do people trust most?',
-    link: 'https://l2beat.com'
-  }
-])
-
-const MenuIcon = {
-  Open: function () {
-    return (
-      <svg
-        width="40"
-        height="24"
-        viewBox="0 0 40 26"
-        fill="none"
-        xmlns="http://www.w3.org/2000/svg"
-      >
-        <line x1="8" y1="1" x2="40" y2="1" stroke="#FFFFFF" strokeWidth="2" />
-        <line x1="8" y1="13" x2="40" y2="13" stroke="#FFFFFF" strokeWidth="2" />
-        <line x1="8" y1="25" x2="40" y2="25" stroke="#FFFFFF" strokeWidth="2" />
-      </svg>
-    )
-  },
-  Close: function () {
-    return (
-      <svg
-        width="24"
-        height="24"
-        viewBox="0 0 29 28"
-        fill="none"
-        xmlns="http://www.w3.org/2000/svg"
-      >
-        <path
-          fillRule="evenodd"
-          clipRule="evenodd"
-          d="M16.8285 14.0005L27.4145 3.4145C28.1965 2.6325 28.1965 1.3685 27.4145 0.5865C26.6325 -0.1955 25.3685 -0.1955 24.5865 0.5865L14.0005 11.1725L3.4145 0.5865C2.6325 -0.1955 1.3685 -0.1955 0.5865 0.5865C-0.1955 1.3685 -0.1955 2.6325 0.5865 3.4145L11.1725 14.0005L0.5865 24.5865C-0.1955 25.3685 -0.1955 26.6325 0.5865 27.4145C0.9765 27.8045 1.4885 28.0005 2.0005 28.0005C2.5125 28.0005 3.0245 27.8045 3.4145 27.4145L14.0005 16.8285L24.5865 27.4145C24.9765 27.8045 25.4885 28.0005 26.0005 28.0005C26.5125 28.0005 27.0245 27.8045 27.4145 27.4145C28.1965 26.6325 28.1965 25.3685 27.4145 24.5865L16.8285 14.0005Z"
-          fill="white"
-        />
-      </svg>
-    )
-  }
-}
-
-function DesktopExternalLink({
-  children,
-  ...props
-}: React.AnchorHTMLAttributes<HTMLAnchorElement>) {
-  return (
-    <ExternalLink
-      className="arb-hover hidden whitespace-nowrap text-base text-white lg:block lg:pr-4"
-      {...props}
-    >
-      {children}
-    </ExternalLink>
-  )
-}
-
-export type HeaderOverridesProps = {
-  imageSrc?: string
-  className?: string
-}
-
-export function HeaderOverrides({ imageSrc, className }: HeaderOverridesProps) {
-  const header = document.getElementById('header')
-
-  if (header) {
-    if (className) {
-      // Reset back to defaults and then add overrides on top of that
-      header.className = defaultHeaderClassName
-      header.classList.add(...className.split(' '))
-    }
-
-    const image = document.getElementById('header-image') as HTMLImageElement
-
-    if (image && imageSrc) {
-      image.src = imageSrc
-    }
-  }
-
-  return null
```

Comment:
conditions are handled within `<Header />` so no extra component is needed for overriding it